<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UICKeyboardShortcutDialog</class>
 <widget class="QWidget" name="UICKeyboardShortcutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>125</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Keyboard shortcuts</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/facetracknoir.png</normaloff>images/facetracknoir.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QCheckBox" name="chkToggleShift">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Shift</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QComboBox" name="cbxToggleKey">
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>0</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Select Number</string>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::InsertAlphabetically</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="textLabel2_5">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Toggle</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QCheckBox" name="chkCenterAlt">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Alt</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QCheckBox" name="chkCenterCtrl">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Ctrl</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QComboBox" name="cbxCenterKey">
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>0</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Select Number</string>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::InsertAlphabetically</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="textLabel2_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Keyboard</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="3" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QCheckBox" name="chkToggleCtrl">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Ctrl</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="textLabel2_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Center</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QCheckBox" name="chkToggleAlt">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Alt</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="chkCenterShift">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Shift</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="ding">
     <property name="text">
      <string>Ding!</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>startEngineClicked()</slot>
  <slot>stopEngineClicked()</slot>
  <slot>cameraSettingsClicked()</slot>
 </slots>
</ui>
